src/api/cached_client.rs

```diff
@@ -35,6 +35,8 @@ pub trait SpotifyApiClient {
 
     fn get_album(&self, id: &str) -> BoxFuture<SpotifyResult<AlbumDescription>>;
 
+    fn get_album_info(&self, id: &str) -> BoxFuture<SpotifyResult<AlbumDescriptionInfo>>;
```

Comment:
Sorry, I should have been clearer; I stand by what I said earlier, there is no need to duplicate the endpoint here; there is no reason get_album and get_album_info would return different things

What I meant when I said that you could have multiple API models was that other models that depend on Album could depend on a different model, for instance (model names are just an example):
- RawSearchResults could have a Option<Page<LightAlbum>> because we don't get copyrights, label, etc
- the regular endpoint would return a FullAlbum
And then those two would be converted to our local model (AlbumDescription) or another one